app/uk/gov/hmrc/pensionschemereturn/transformations/nonsipp/MemberPaymentsTransformer.scala

```diff
@@ -63,7 +63,12 @@ class MemberPaymentsTransformer @Inject()(
       schemeReceivedTransferIn = memberPayments.transfersInMade,
       schemeMadeTransferOut = memberPayments.transfersOutMade,
       lumpSumReceived = memberPayments.lumpSumReceived,
-      pensionReceived = Option.when(memberPayments.pensionReceived.started)(memberPayments.pensionReceived.made),
+      pensionReceived = memberPayments.pensionReceived match {
+        case SectionDetails(made @ true, completed @ true) => Some(true)
+        case SectionDetails(made @ true, completed @ false) => Some(true)
+        case SectionDetails(made @ false, completed @ true) => Some(false)
+        case SectionDetails(made @ false, completed @ false) => None // not started
+      },
```

Comment:
I'm using the same logic as for Surrendered Benefits, except that the 2nd case returns Some(true) rather than Some(false). This is because Some(false) is incorrect for this case now that the statuses have been redesigned, it's just that Surrendered Benefits is being fixed in a different ticket that's currently in progress.

Reply:
Do we need a test for this?

Reply:
Yes, thanks for the reminder - it seems like there's currently only tests for Surrendered Benefits, Employer Contributions, and Member Details in `MemberPaymentsTransformerSpec`, so I'll create tests for Pension Payments on this branch and let you know once I've pushed them.